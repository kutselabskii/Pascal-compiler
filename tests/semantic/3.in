program abc;

type a = integer;

var a: integer;

begin end
.