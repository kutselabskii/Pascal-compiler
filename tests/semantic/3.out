Line 5 : Col 6 : Semantic error :: Id duplication
